force-app/main/default/classes/ProgramPlanSettingsMapper_TEST.cls

```diff
@@ -0,0 +1,112 @@
+            programPlanSettingsMapperInstance,
+            'Instance of mapper class returned should match static instance variable.'
+        );
+        System.assertEquals(
+            true,
+            programPlanSettingsMapperInstance != null,
+            'Instance of mapper class should not be null.'
+        );
+    }
+
+    /**************************************************************************************************************************
+     * @description Test method to verify that the getInstance method returns the existing instance of the
+     * programPlanSettingsMapperInstance class when one already exists.
+     ***************************************************************************************************************************/
+    @isTest
+    private static void getInstanceExisting() {
+        Test.startTest();
+        ProgramPlanSettingsMapper programPlanSettingsMapperInstance1 = ProgramPlanSettingsMapper.getInstance();
+        ProgramPlanSettingsMapper programPlanSettingsMapperInstance2 = ProgramPlanSettingsMapper.getInstance();
+        Test.stopTest();
+
+        System.assertEquals(
+            ProgramPlanSettingsMapper.instance,
+            programPlanSettingsMapperInstance1,
+            'Instance of mapper class returned should match static instance variable.'
+        );
+        System.assertEquals(
+            programPlanSettingsMapperInstance1,
+            programPlanSettingsMapperInstance1,
+            'Subsequent retrievals of mapper class instance should return existing instance.'
+        );
+        System.assertEquals(
+            true,
+            programPlanSettingsMapperInstance1 != null,
+            'Instance of mapper class should not be null.'
+        );
+    }
+
+    /**************************************************************************************************************************
+     * @description Test method to verify that the getProgramPlanSettingsModel method returns a
+     * ProgramPlanSettingsModel matching the program plan settings specified in hierarchy
+     * settings.
+     ***************************************************************************************************************************/
+    @isTest
+    private static void getProgramPlanSettingsModelValid() {
+        Hierarchy_Settings__c hierarchySettings = UTIL_CustomSettings_API.getSettingsForTests(
+            new Hierarchy_Settings__c(Validate_Program_Plan_for_Nested_PR__c = true)
+        );
+
+        ProgramPlanSettingsMapper programPlanSettingsMapperInstance = ProgramPlanSettingsMapper.getInstance();
+
+        Test.startTest();
+        ProgramPlanSettingsModel ppSettingsModel = programPlanSettingsMapperInstance.getProgramPlanSettingsModel();
+        Test.stopTest();
+
+        System.assertEquals(
+            true,
+            ppSettingsModel.validateProgramPlanForNestedPR,
+            'validateProgramPlanForNestedPR should not be enabled.'
```

Comment:
Should this be "enabled" instead of"not be enabled"?

Reply:
I fixed the text